src/public/services/__tests__/AdminService.test.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import MockAxios from 'jest-mock-axios'
+import { UiCookieValues } from 'shared/types'
+
+import { adminChooseEnvironment } from '../AdminService'
+
+jest.mock('axios', () => MockAxios)
+
+describe('AdminService', () => {
+  describe('adminChooseEnvironment', () => {
+    const MOCK_UI = UiCookieValues.React
+    const MOCK_RESPONSE = UiCookieValues.React
+
+    it('should return environment name if GET request succeeds', async () => {
+      // Arrange
+      const mockUi = UiCookieValues.React
+      const expectedResponse = UiCookieValues.React
+
+      // Act
+      const actual = adminChooseEnvironment(MOCK_UI)
+      MockAxios.mockResponse({ data: MOCK_RESPONSE })
+
+      // Assert
+      await expect(actual).resolves.toEqual(MOCK_RESPONSE)
+      expect(MockAxios.get).toHaveBeenCalledWith(
+        `/api/v3/admin/environment/${MOCK_UI}`,
+        expectedResponse,
+        { params: { data: mockUi } },
```

Comment:
How are these arguments part of the axios `get()` call 🤔 ? The call in AdminService only supplies the URL, no?

Reply:
you're right, have removed!